sourcegraph/jetbrains
# Update color scheme for cody tool window icon

Fixes https://github.com/sourcegraph/sourcegraph/issues/57293

This PR updates color scheme for Cody tool window icon. 
It now matches IntelliJ guidelines:
https://plugins.jetbrains.com/docs/intellij/work-with-icons-and-images.html#new-ui-icon-colors

Btw it fixes JetBrains: Cody icon in toolbar looks weird when cody is selected (linked issue).

## Screenshots

1. Icon selected, dark theme
![image](https://github.com/sourcegraph/jetbrains/assets/19799111/fcb4ff49-1fa4-4197-bbf4-8ba38a554a9a)

2. Icon selected, dark theme, but no focus on the button
![image](https://github.com/sourcegraph/jetbrains/assets/19799111/12f15c65-2888-45ab-8283-1f71f41cb98a)

3. Some other icon selected
![image](https://github.com/sourcegraph/jetbrains/assets/19799111/5dc99010-4c9f-4e8d-b356-4b4d222d27f8)

I verified it for the light theme too - works as it should!

## Test Plan

1. check out my branch
2. build the plugin
3. install the plugin to IntelliJ with new ui turned on 
4. verify
